sdk/core/core-rest-pipeline/src/defaultHttpClient.browser.ts

```diff
@@ -2,11 +2,11 @@
 // Licensed under the MIT license.
 
 import { HttpClient } from "./interfaces";
-import { createXhrHttpClient } from "./xhrHttpClient";
```

Comment:
Do we still allow a user to create an XHR client or is `createXhrHttpClient` now dead code?

Reply:
I think it is only being used in tests? @xirzec should that function be moved to the tests?

Reply:
We never publicly exposed creating the client directly, so yeah I moved this wholesale into test-utils (GitHub is showing it as a rename)